src/mcp/client.py

```
@@ -72,33 +134,125 @@ async def _get_server_tools(self, server_name: str) -> list[BaseTool]:
+                        self._load_live_tool,
+                    )
+                    self._apply_metadata(lazy_tool)
+                    tools.append(lazy_tool)
+
+            if pending_servers:
+                results = await asyncio.gather(
+                    *[self._load_and_cache_server(s) for s in pending_servers],
+                    return_exceptions=True,
+                )
+
+                for server_name, server_result in zip(pending_servers, results):
```

Ruff (0.14.6)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)